cloudbuild.yaml

```diff
@@ -1,119 +1,87 @@
+    docker tag "gcr.io/cloud-marketplace-tools/k8s/dev:$$TAG" "gcr.io/cloud-marketplace-tools/k8s/dev:build-$BUILD_ID"
 
-- id: Initialized
-  name: bash
+- id: &PullEnvsubstImage Pull Envsubst Image
+  name: gcr.io/cloud-builders/docker
   waitFor:
-  - Initialize git
-  - Get Kubernetes Credentials
-  - Pull Dev Image
-
-# Wordpress Verification
-
-- id: Fetch Wordpress Reporting Secret
-  name: gcr.io/cloud-builders/gsutil
+  - '-'
+  entrypoint: bash
   args:
-  - cp
-  - gs://cloud-marketplace-ops-test-kokoro/reporting_secrets/wordpress.yaml
-  - reporting-secret.json
+  - -exc
+  - |
+    TAG="$$(cat MARKETPLACE_TOOLS_TAG)"
+    docker pull "gcr.io/cloud-marketplace-tools/k8s/deployer_envsubst:$$TAG"
+    docker tag "gcr.io/cloud-marketplace-tools/k8s/deployer_envsubst:$$TAG" "gcr.io/cloud-marketplace-tools/k8s/deployer_envsubst:build-$BUILD_ID"
 
-- id: Build Wordpress
-  name: gcr.io/cloud-marketplace-tools/k8s/dev:local
+- id: &PullHelmTillerImage Pull Helm Tiller Image
+  name: gcr.io/cloud-builders/docker
   waitFor:
-  - Initialized
-  dir: wordpress
+  - '-'
+  entrypoint: bash
   args:
   - -exc
-  - make -j4 clean app/build
+  - |
+    TAG="$$(cat MARKETPLACE_TOOLS_TAG)"
+    docker pull "gcr.io/cloud-marketplace-tools/k8s/deployer_helm_tiller/onbuild:$$TAG"
+    docker tag "gcr.io/cloud-marketplace-tools/k8s/deployer_helm_tiller/onbuild:$$TAG" "gcr.io/cloud-marketplace-tools/k8s/deployer_helm_tiller/onbuild:build-$BUILD_ID"
 
-- id: Verify Wordpress
-  name: gcr.io/cloud-marketplace-tools/k8s/dev:local
+- id: &InitializeCredentials Initialize Credentials
+  name: gcr.io/cloud-builders/gcloud
   waitFor:
-  - Initialized
-  - Build Wordpress
-  - Fetch Wordpress Reporting Secret
-  env:
-  - APP_DEPLOYER=gcr.io/cloud-marketplace-ops-test/example/wordpress/deployer:latest
-  - |
-    APP_PARAMETERS={
-                     "reportingSecret": "file://reporting-secret.json",
-                     "imageTester": "gcr.io/cloud-marketplace-ops-test/example/wordpress/tester:latest"
-                   }
+  - '-'
+  entrypoint: bash
   args:
   - -exc
   - |
-    /scripts/verify \
-        --deployer="$$APP_DEPLOYER" \
-        --parameters="$$APP_PARAMETERS"
+    gcloud container clusters get-credentials 'limani-integ' --region 'us-central1' --project '$PROJECT_ID'
+    mkdir -p /workspace/.kube/
+    cp -r $$HOME/.kube/ /workspace/
+    mkdir -p /workspace/.config/gcloud/
+    cp -r $$HOME/.config/gcloud/ /workspace/.config/
 
-# NGINX Verification
-
-- id: Build NGINX
-  name: gcr.io/cloud-marketplace-tools/k8s/dev:local
+- id: Build and Verify helm/
+  name: gcr.io/cloud-marketplace-tools/k8s/dev:build-$BUILD_ID
   waitFor:
-  - Initialized
-  dir: nginx
+  - *PullDevImage
+  - *PullHelmTillerImage
+  - *InitializeCredentials
+  env:
+  - 'MARKETPLACE_TOOLS_TAG=build-$BUILD_ID'
+  - 'KUBE_CONFIG=/workspace/.kube'
+  - 'GCLOUD_CONFIG=/workspace/.config/gcloud'
+  - 'EXTRA_DOCKER_PARAMS=--net cloudbuild'
+  entrypoint: bash
```

Comment:
not sure how this could have worked. The `dev` container has an entrypoint that sets up the kubeconfig and what not. I think what you mean to do here is:

```yaml
env: [...]
args:
- bash
- -exc
- |
  helm/deployer build
  helm/deployer verify
```